uk_bin_collection/uk_bin_collection/councils/MidlothianCouncil.py

```
@@ -1,178 +1,158 @@
+                    or d.find_element(By.ID, "dateRecycling").get_attribute("value")
+                    or d.find_element(By.ID, "dateFood").get_attribute("value")
+                )
+            )
+            time.sleep(1)
+
+            bins = []
+            for field_id, bin_type in self.DATE_FIELDS.items():
+                try:
+                    value = driver.find_element(By.ID, field_id).get_attribute("value")
+                except Exception:
+                    continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)